### ISSUE TITLE
allow match on cohortid

### ISSUE THREAD

**Issue by danmarsden** [catalyst/moodle-local_cohortauto#18]
In some cases it would be useful to allow the admin to modify the cohort name after it is created - we should look at using the idnumber field for matching the data so that the display name of the cohort can change.

**Comment by cjbsdigitallearning** [comment]
I agree with this it seems a little counter intuitive that the ID is not set, this means amongst other things that you have to add an ID is you want to be able to filter by cohort in the user list.  If this feature does get added it would be great to have an option to set the name based on the ID by default and to have the replace function act on both the ID and the name.
